tss-esapi-sys/build.rs

```diff
@@ -7,18 +7,111 @@ fn main() {
         return;
     }
 
+    #[cfg(feature = "bundled")]
+    let source_path;
+    #[cfg(feature = "bundled")]
+    {
+        use std::io::Write;
+        let out_path = std::env::var("OUT_DIR").expect("No output directory given");
+        source_path = fetch_source(
+            out_path,
+            "tpm2-tss",
+            "https://github.com/tpm2-software/tpm2-tss.git",
```

Comment:
I do not think it is necessary to expose the address and name like this. I think it would make more sense to move `fetch_source` into the `tpm2_tss` module instead of having it generic like this.